### PR TITLE
Add Nayru's Love to minimal item pool on high damage

### PR DESCRIPTION
**Pull request by Star1468**
Currently, Nayru's Love is completely removed from the item pool when set to minimal. This is fine on most damage multipliers and suits the nature of minimal item pool quite well, but can be problematic on OHKO and quad damage, something that was noticed during a random settings league match. Also fixing a comment that seems to have been copy pasted from ALTTPR code that I noticed and was bugging me.

### CODE REVIEW

**Comment by LostThePirate** [comment]
Fixed pendants/crystals to stones/medallions, but left the reference to using ALttP's medallions for entering MM and TR. ;)

**Comment by Star1468** [comment]
Ah, brain farted what that comment was referring to and didn't want to delete it by accident. Fixed now.

**Comment by Hamsda** [comment]
I think that is an old relic from when we introduced the different item pools.
Back then there were no damage multipliers, so the pools were kind of intended for difficulty as well. Minimal not having NL was to avoid just cheesing combat by using NL, but nowadays I think this is definitely the better solution.

**Comment by Star1468** [comment]
That makes a lot of sense. The possibility of cheesing combat with NL in the lower damage multipliers is why I didn't just permanently re-add NL to the pool.

**Review by Zannick — APPROVED** [review]

File: ItemPool.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 #This file sets the item pools for various modes. Timed modes and triforce hunt are enforced first, and then extra items are specified per mode to fill in the remaining space.
-#Some basic items that various modes require are placed here, including pendants and crystals. Medallion requirements for the two relevant entrances are also decided.
+#Some basic items that various modes require are placed here, including stones and medallions.
```
Comment (Roman971) [inline ItemPool.py:13]: Changing this comment isn't allowed D:

In all seriousness tho, this comment is completely wrong and is just a leftover from alttpr, so if we're going to touch it, it should just be removed entirely imo. (Aside from maybe setting the item pools, but that's pretty obvious from the name of the file and functions)

File: ItemPool.py [hunk 2]
```diff
@@ -1323,6 +1323,9 @@ def get_pool_core(world):
     for item,max in item_difficulty_max[world.item_pool_value].items():
         replace_max_item(pool, item, max)
 
+    if world.damage_multiplier in ['ohko', 'quadruple'] and world.item_pool_value == 'minimal':
+        pending_junk_pool.append ('Nayrus Love')
```
Comment (Roman971) [inline ItemPool.py:1327]: Minor formatting issue
```suggestion
        pending_junk_pool.append('Nayrus Love')
```

Comment (Zannick) [reply ItemPool.py:1327]: Done.

**Pull request merged**